src/main/kotlin/org/openrewrite/kotlin/internal/KotlinParserVisitor.kt

```diff
@@ -1393,7 +1393,8 @@ class KotlinParserVisitor(
                 prefix,
                 Markers.EMPTY,
                 variableDeclarations,
-                JContainer.build(before, vars, Markers.EMPTY)
+                // JContainer.build(before, vars, Markers.EMPTY)
+                JContainer.build(before, emptyList(), Markers.EMPTY) // pass compile
```

Comment:
Ah, I didn't think this still existed -- could you remove the `KotlinParserVisitor` in a separate PR?

Reply:
Sure, why not.